app/presenters/event_presenter.rb

```diff
@@ -0,0 +1,74 @@
+class EventPresenter < BasePresenter
+  presents :event
+
+  delegate :day, :month, :day_of_the_month, :hour, to: :date_presenter
+
+  def speakers_css_class
+    extended_event? ? 'pk-main-speakers--four-speakers' : ''
+  end
+
+  def talks
+    @talk ||= event.talks.includes(:speaker)
+  end
+
+  def show_talks?
+    talks.any?
+  end
+
+  def action_button
+    return confiration_partial if confirmation_allowed?
+
+    if current_user
+      authorized_variant
+    else
+      { partial: 'events/unauthorized' }
+    end
+  end
+
+  # Question: Is it used somewhere?
+  #
+  def attendees
```

Comment:
specs should give an answer. If it does not fail after method removal than no.

Reply:
Yeah, Thanks =). Got It.